### DISCUSSION TITLE
Navbar hides few elements in mobile browser view (iphone 12 safari)

### DISCUSSION THREAD

**Discussion by dpetrishin** [requarks/wiki#3333] [Error / Bug Report]
**Describe the bug**
When I open any page on wiki being logged it as admin in iphone 12 - few elements are hidden and not possible to be clicked.

**To Reproduce**
1. Login to wiki as admin from mobile (iphone 12)
2. Open any page
3. Few Navbar elements are not able to be clicked

**Expected behavior**
To have collapsed button to show all navbar elements/fit elements on navbar

**Screenshots**
![photo_2021-02-24_09-45-45](https://user-images.githubusercontent.com/28645748/108973821-1893cf00-7685-11eb-8bc7-311fa47ef25d.jpg)

**Host Info (please complete the following information):**
 - OS: [Docker]
 - Wiki.js version: [2.5.170]
 - Database engine: latest

**Comment by ZeroGravitas23** [comment]
I noticed this two on multiple page types.

First is the top navigation bar which on my phone seems to be at least hiding the ability to search for tags on a mobile browser:
![Main Screen Navbar issue](https://user-images.githubusercontent.com/80851921/111539558-0107ad80-873c-11eb-94bb-d9bffffce159.jpg)

Also, I am seeing the issue when editing a page on mobile. It doesn't show all editing elements like the options for bulleted & numbered lists:
![Editing Page Navbar issue](https://user-images.githubusercontent.com/80851921/111539864-63f94480-873c-11eb-9569-1a64e6274fa1.jpg)

**Host Info (please complete the following information):**

OS: [Docker]
Wiki.js version: [2.5.170]
Database engine: latest

**Reply by ZeroGravitas23** [reply]
Any update on this? Or is this something that will be fixed in version 3?
